static/js/lib/embed_test.js

```diff
@@ -48,4 +58,59 @@ describe("embed utils", () => {
     assert.equal(link.getAttribute("href"), url)
     assert.equal(link.getAttribute("class"), "embedly-card")
   })
+
+  describe("initialization", () => {
+    let embedlyStub
+
+    beforeEach(() => {
+      document.head.innerHTML = ""
+      document.body.innerHTML = ""
+
+      // this expects at least one script element to already exist
+      embedlyStub = sandbox.stub()
+      global.embedly = embedlyStub
+
+      const script = document.createElement("script")
+      document.head.appendChild(script)
+    })
+
+    it("loads embedly", () => {
+      loadEmbedlyPlatform()
+      assert.equal(document.querySelectorAll("script").length, 2)
+      const firstScript = document.querySelectorAll("script")[0]
+      assert.equal(
+        firstScript.getAttribute("src"),
+        "http://cdn.embedly.com/widgets/platform.js"
+      )
+    })
+    ;[true, false].forEach(hidesTitle => {
+      it(`${
+        hidesTitle ? "adds" : "doesn't add"
+      } a style element to hide the title`, () => {
+        loadEmbedlyPlatform()
+
+        const container = document.createElement("div")
+        document.body.append(container)
+        if (hidesTitle) {
+          container.setAttribute("class", "no-embedly-title")
+        }
+        const iframe = document.createElement("iframe")
+        container.appendChild(iframe)
+
+        const callback = embedlyStub.firstCall.args[2]
```

Comment:
it might be just a little easier to follow what's going on here if the callback were defined as a separate function in `lib/embed.js`, so you don't have to grab it out of the stub here

Reply:
I think I would need to export the function then reimport it in order to be able to stub it as a separate function. Does that sound right?

Reply:
not sure I follow - I was proposing that the function defined inline on line 63 of `lib/embed.js` be defined in that file as a separate function just to make testing a little easier. I don't think that would require any changes to how you're stubbing things, unless I'm missing something.

Reply:
Ah, I think I misunderstood. I made the change in the last commit, let me know if that's what you had in mind.